mitiq/calibration/settings.py

```diff
@@ -208,29 +208,34 @@ def make_problems(self) -> List[BenchmarkProblem]:
         Returns:
             A list of :class:`BenchmarkProblem` objects"""
         circuits = []
-        nqubits, depth, seed = (
-            self.num_qubits,
-            self.circuit_depth,
-            self.circuit_seed,
-        )
-        for i, circuit_type in enumerate(self.circuit_types):
+        for i, benchmark in enumerate(self.benchmarks):
+            circuit_type = benchmark["circuit_type"]
+            num_qubits = benchmark["num_qubits"]
+            if (
+                "circuit_depth" in benchmark.keys()
+                and benchmark["circuit_depth"] is not None
+            ):
+                depth = benchmark["circuit_depth"]
+            else:
+                depth = num_qubits
```

Comment:
Could we simplify this by using [`dict.get`](https://docs.python.org/3/library/stdtypes.html?highlight=get#dict.get)?
```suggestion
            depth = benchmark.get("circuit_depth")
```

Reply:
Yes, good catch!